src/main/scala/jigg/pipeline/Pipeline.scala

```diff
@@ -148,15 +150,13 @@ Currently the annotators listed below are installed. See the detail of each anno
         annotatorList.foldLeft(RequirementSet()) { (satisifedSofar, annotator) =>
           try annotator.checkRequirements(satisifedSofar)
           catch { case e: RequirementError =>
-            println("***** WARNING *****")
-            println(e.getMessage)
-            println("*******************")
+            System.out.println(s"[warn] %s".format(e.getMessage))
```

Comment:
Sorry for comments on previous commit. Could you remove duplications in code by higher-order function? E.g.,

```scala
    def check(procError: (RequirementError, RequirementSet)=>RequirementSet) =
      annotatorList.foldLeft(RequirementSet()) { (satisfiedSoFar, annotator) =>
        try annotator.checkRequirements(satisfiedSoFar)
        catch { case e: RequirementError => procError(e, satisfiedSoFar) }
      }
    
    checkRequirement match {
      case "true" => check { (e, satisfiedSoFar) =>
        argumentError("annotators", e.getMessage)
      }
      case "warn" => check { (e, satisfiedSoFar) =>
        System.out.println(s"[warn] %s".format(e.getMessage))
        satisfiedSoFar
      }
...
```
This may also highlight the difference between two options.